app/scripts/slext.ts

```diff
@@ -40,17 +40,61 @@ export class Slext extends Dispatcher {
         return $(".full-size.ng-scope:not(.ng-hide)[ng-show=\"ui.view == 'pdf'\"]").length > 0;
     }
 
+    public isFullScreenEditor(): boolean {
+        return !this.isFullScreenPDF() && !this.isSplitScreen();
+    }
+
     public isHistoryOpen(): boolean {
         return $("#ide-body.ide-history-open").length > 0;
     }
 
-    public goToFullScreenPDF(): void {
+    private _toggleFullScreenPDFEditor(): void {
         const button = $('[ng-click="togglePdfView()"]');
         if (button.length) {
             (button[0] as HTMLElement).click();
         }
     }
```

Comment:
Upon testing this does not seem to work. It seems we can use `.toolbar-pdf-expand-btn` instead however.

Reply:
Hi Mathias! I tested it again just now on both Firefox and Chrome and fount it working. Also I did not find ``.toolbar-pdf-expand-btn``.

Reply:
Interesting. I did some digging and found that the button is changed in the beta programme at the moment (and will rolled out to everyone in the near future I'd assume) It was introduced in this commit https://github.com/overleaf/web/commit/df0b53592a570489dcb1f015053baa2b4c1602cd#diff-f2561b1a9b6df13066f20ff8424aea7106f8e05c699c7e5c603897f455c09df0R246 which is currently in the "preview features" directory. 

So the best thing would be to support both elements. I can have a go at this since I can test it.

Reply:
I did this in ed821b6b07007279f6cd5c126e469fff01211792 and then everything worked.